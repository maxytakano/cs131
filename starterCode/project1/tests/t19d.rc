structdef MS { 
     int a;
     int b; 
     function : int doo() { return 0; } // has no effect on size! 
}; 

int x; 
const float y = 55.5; 
bool z[4]; 
MS t;

function : void main() { 
     foo(z, z); 
     x = sizeof(float); // should be 4 
     x = sizeof(MS);  // should be 8 
     x = sizeof(float***); // should be 4 
     x = sizeof(int[3]); // should be 12 
     x = sizeof(x);  // should be 4 
     x = sizeof(y);  // should be 4 
     x = sizeof(z);  // should be 16 
     x = sizeof(t);  // should be 8 
} 
