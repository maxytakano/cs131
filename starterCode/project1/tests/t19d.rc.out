Error, "/tmp/cs131_testrunner__ITaux/file.rc", line 13: 
  undeclared identifier 'foo'.
Compile: failure.
